winterfell/webapp/src/js/profileService.js

```diff
@@ -1,12 +1,47 @@
 'use strict';
 var app = angular.module('vetafiApp');
-app.factory('profileService', ['claimService', function(claimService) {
-  return {
-    userInfo: {},
-    clearUserInfo: function() {
-      sessionStorageHelper.removePair(vfiConstants.keyUserId);
-      this.userInfo = {};
+app.service('Profile', ['claimService', '$q', 'net', function(claimService, $q, net) {
```

Comment:
I'm not sure what you mean, this is the only profileService, i replaced the old one, and changed it to be `app.service` so that it is a singleton